modules/tensor_mechanics/src/materials/ComputeLinearElasticStress_CUT_DT.C

```diff
@@ -0,0 +1,50 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ComputeLinearElasticStress_CUT_DT.h"
+
+template<>
+InputParameters validParams<ComputeLinearElasticStress_CUT_DT>()
+{
+  InputParameters params = validParams<ComputeStressBase>();
+  params.addClassDescription("Compute stress using elasticity for small strains");
+  return params;
```

Comment:
Right, this was just @WilkAndy's test code so we can change the name, etc.  What name would you prefer?  There's currently no place in the modules for test-specific code, otherwise this should really go there.